evap/results/templates/results_index.html

```diff
@@ -128,6 +128,15 @@ <h6 class="card-subtitle mb-1">{% trans 'Semester' %}</h6>
             });
         };
 
+        function includesAny(string, array){
+            for(var i = 0; i < array.length; i++){
+                if(string.includes(array[i])){
+                    return true;
+                }
+            }
+            return false;
```

Comment:
this would be equivalent to `return array.some(s => searchstring.includes(s))`

but actually, don't you want `s.includes(searchstring)`?

Reply:
i was looking for something like `some`, thanks :)

the searchstring consists of all selected degrees. so i want to check whether the array contains any element that is included in the string.

Reply:
ah, right. i assumed this was something the user entered manually.

Reply:
please review again